coding_style/README.adoc

```diff
@@ -150,6 +150,16 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   But do use quotes for user-side strings such as descriptions, names, and messages.
 * Even if JSON is valid YAML and Ansible understands it, do only use JSON syntax if it makes sense (e.g. a variable file automatically generated) or adds to the readability.
   In doubt, nobody expects JSON so stick to YAML.
+* A single jinja template should not exceed 250 lines in length
```

Comment:
I would create one single recommendation about keeping Jinja2 structures simple enough, and then have the bullet points become part of the explanation.

Reply:
On this specific line:
1. I'm still struggling with the specific number
2. nobody creates a 250 lines Jinja template for fun (well, obfuscated Ansible contest perhaps?), I can't even imagine a good use case. But if someone would come with a reasonable use case, what would be the alternative approach?

Reply:
Agreed @ericzolf. This is another case of an arbitrary number I am opposed to having in these guidelines. If I have a large configuration file example from an ISV I need to template I should have to cut it up into piece just to reassemble it using includes order to comply here. All of these guidelines should focus the most common cases on providing a baseline that can be checked by tooling like ansible-lint.

Reply:
+1  We also discussed in another PR and I'll remove all recommendations for hard numbers